parsec/core/fs/storage/manifest_storage.py

```diff
@@ -88,6 +92,47 @@ async def _create_db(self):
                 );
                 """
             )
+            # Singleton storing the pattern_filter
+            cursor.execute(
+                """
+                CREATE TABLE IF NOT EXISTS pattern_filter
+                (
+                  _id INTEGER PRIMARY KEY NOT NULL,
+                  pattern TEXT NOT NULL,
+                  fully_applied INTEGER NOT NULL  -- Boolean
+                );
+                """
+            )
+            # Set the default pattern filter if it doesn't exist
+            cursor.execute(
+                """
+                INSERT OR IGNORE INTO pattern_filter(_id, pattern, fully_applied)
+                VALUES (0, ?, 0)""",
+                (EMPTY_PATTERN,),
```

Comment:
I guess it's better to store the empty pattern in database than return it from the code if the SELECT in `get_pattern_filter` returns no row given we have the guarantee the returned pattern is the one that has been used (and not something else given we are running a new version of Parsec where `EMPTY_PATTERN` has changed)